### PR TITLE
Fix Overlapping issues in FAQ Mentor/Mentee Section

### LINKED ISSUE #1428: Overlaping issues in ScholarX page (Smartphones)
**Describe the bug**
There are overlapping issues in FAQ for Mentees/Mentor sections when you use smartphones to view the section

**To Reproduce**
1. Go to 'ScholarX page'
2. Scroll down to 'FAQ for Mentees/Mentor' section (Issue is visible for some smartphones.)

**Expected behavior**
Able to view that section using any smartphone.

**Screenshots**
overlap

**Additional context**
After fixing the bug.
image

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by janithlahirukariyawasam**
## Purpose

The purpose of this PR is to fix #1428 

## Goals

Fixed Overlapping issues in FAQ Mentor/Mentee Section.
Faced when you view the section using smartphones.

### Screenshots

**_ERROR_**
overlap

overlap pull issue

**_AFTER FIXING_**
pull overlapping fix 2
pull overlap fix

  
### Preview Link

https://pr-1429-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/2022/index.html [hunk 1]
```diff
@@ -210,12 +210,19 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentors</h1>
...
-                                    </div>
-                                    <div class="text-primary collapse-sign">
-                                        <i class="plusMinusSign fa fa-plus float-right"></i>
-                                    </div>
+                                        <table class="faq-table">
+                                            <tr>
+                                                <td>
+                                                    <div class="question-heading">
+                                                        What are the eligibility 
+                                                    </div>requirements?
```
Comment (YoshithaRathnayake) [inline scholarx/2022/index.html]: Can you put this `requirements?` text inside the div tag with `What are the eligibility` ? @janithlahirukariyawasam If there a reason for putting it outside the div tag let me know.

File: scholarx/2022/index.html [hunk 2]
```diff
@@ -401,14 +449,21 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentees</h1>
...
-                                        <div class="text-primary collapse-sign"><i
-                                                class="plusMinusSign fa fa-plus float-right"></i>
-                                        </div>
+                                            aria-controls="collapseFive">   
+                                        <table class="faq-table">
+                                            <tr>
+                                                <td>
+                                                    <div class="question-heading">
+                                                        What are the eligibility
+                                                    </div>requirements?
```
Comment (YoshithaRathnayake) [inline scholarx/2022/index.html]: Here too @janithlahirukariyawasam

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: scholarx/2022/index.html (hunk 2 above)
Comment (janithlahirukariyawasam) [reply scholarx/2022/index.html]: @YoshithaRathnayake Can you check now?

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html]: Great!

**Comment by janithlahirukariyawasam** [comment]
@YoshithaRathnayake Wait I will explain the reason, So don't merge

**Comment by janithlahirukariyawasam** [comment]
image

I had to add the "requiremetns" in a new line always that was the reason
If not @YoshithaRathnayake If you see the section using very thin displays like Galaxy Fold the " + " mark will not align correctly

**Comment by YoshithaRathnayake** [comment]
image

But the thing is, when we watch it on desktop devices it is broken into two lines. Can you avoid it? @janithlahirukariyawasam If you can, its great.

**Comment by janithlahirukariyawasam** [comment]
image

@YoshithaRathnayake This is the issue, Actually the error is only **visible in very thin displays.** **( 280 )**
" + " isn't align with others, 
So what your opinion? I think it is quit hard to fix. So may I put "requirement" inside the div tag?

**Work perfectly with all displays except very thin like new galaxy fold**

File: scholarx/2022/index.html (hunk 1 above)
Comment (YoshithaRathnayake) [inline scholarx/2022/index.html:218]: Ok then for now put this inside the div tag and separate them with a `<br>` tag @janithlahirukariyawasam

File: scholarx/2022/index.html [hunk 3]
```diff
@@ -401,14 +449,21 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentees</h1>
...
-                                        <div class="text-primary collapse-sign"><i
-                                                class="plusMinusSign fa fa-plus float-right"></i>
-                                        </div>
+                                            aria-controls="collapseFive">   
+                                        <table class="faq-table">
+                                            <tr>
+                                                <td>
+                                                    <div class="question-heading">
+                                                        What are the eligibility 
+                                                    </div>requirements?
```
Comment (YoshithaRathnayake) [inline scholarx/2022/index.html:458]: Here too @janithlahirukariyawasam

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: scholarx/2022/index.html (hunk 1 above)
Comment (YoshithaRathnayake) [inline scholarx/2022/index.html]: Ok then for now add the `requirements?` text inside the div tag and separate them with a `<br>` tag @janithlahirukariyawasam

File: scholarx/2022/index.html (hunk 3 above)
Comment (YoshithaRathnayake) [inline scholarx/2022/index.html]: Here too @janithlahirukariyawasam

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: scholarx/2022/index.html (hunk 3 above)
Comment (janithlahirukariyawasam) [reply scholarx/2022/index.html]: **With "<br" tag** 
image
in div br

**Outside the div**
image

error not align out div


**Issue is visible to the thin displays with that implementation also.**

**Comment by YoshithaRathnayake** [comment]
A solution found @janithlahirukariyawasam

File: scholarx/2022/index.html [hunk 4]
```diff
@@ -210,12 +210,19 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentors</h1>
                                         data-target="#collapseTwo"
                                         aria-expanded="false"
                                         aria-controls="collapseTwo">
-                                    <div class="question-heading">
-                                        What are the eligibility requirements?
-                                    </div>
-                                    <div class="text-primary collapse-sign">
-                                        <i class="plusMinusSign fa fa-plus float-right"></i>
-                                    </div>
+                                        <table class="faq-table">
```
Comment (YoshithaRathnayake) [inline scholarx/2022/index.html]: Add `mr-4` class to the overlapping questions without this table @janithlahirukariyawasam. Try it and let me know whether it is working. I tried its worked.

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: scholarx/2022/index.html (hunk 4 above)
Comment (YoshithaRathnayake) [reply scholarx/2022/index.html]: It should definitely work @janithlahirukariyawasam

Comment (janithlahirukariyawasam) [reply scholarx/2022/index.html]: @YoshithaRathnayake please if you can show the implementation of that using screenshot?

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html]: Do you want a screenshot of the code right? @janithlahirukariyawasam

Comment (janithlahirukariyawasam) [reply scholarx/2022/index.html]: Yes @YoshithaRathnayake

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html]: image

Remove the table and add the div tags like earlier @janithlahirukariyawasam

Comment (janithlahirukariyawasam) [reply scholarx/2022/index.html]: Thank you, I will try this and inform you.

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html]: Sure

Comment (janithlahirukariyawasam) [reply scholarx/2022/index.html]: @YoshithaRathnayake Yes Thank you very much!
It solves the issue.

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html]: Your welcome @janithlahirukariyawasam

Comment (janithlahirukariyawasam) [reply scholarx/2022/index.html]: @YoshithaRathnayake Can you check now?
Thank you very very much again.

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html]: Sure

File: assets/css/global-custom.css [hunk 5]
```diff
@@ -477,6 +477,10 @@ body {
     padding-right: 30px;
 }
 
+.faq-table{
```
Comment (YoshithaRathnayake) [inline assets/css/global-custom.css]: We don't want this right? Remove this @janithlahirukariyawasam

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/global-custom.css (hunk 5 above)
Comment (janithlahirukariyawasam) [reply assets/css/global-custom.css]: Ah yes!

Comment (janithlahirukariyawasam) [reply assets/css/global-custom.css]: Thank you again Master Mind!

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job !
Thank You @janithlahirukariyawasam

File: assets/css/global-custom.css (hunk 5 above)
Comment (YoshithaRathnayake) [reply assets/css/global-custom.css]: Any time @janithlahirukariyawasam

**Pull request merged**